### DISCUSSION TITLE
Is the facetted searching released in MeiliSearch to support wildcard search

### DISCUSSION THREAD

**Discussion by ravisharmaa** [meilisearch/product#344] [Feedback & Feature Proposal]
Hey @NoahSaso,

First of all, thank you for your work! That is really clean!

I am wondering if this is a feature we want to add to the engine:
  1. It could impact the speed of the search too much.
  2. We will release facetted searching (aggregations) that will solve the use case you probably have.
  3. Introducing a new operator is probably not that good.

Anyway, I would like too ear more about the use case you have, please, just to see if another feature could fit you.

_Originally posted by @Kerollmops in https://github.com/meilisearch/MeiliSearch/issues/610#issuecomment-615358792_

**Comment by curquiza** [comment]
Hello @ravisharmaa, I moved your issue into the right repo so that the product team can consider it :)

**Comment by gmourier** [comment]
Closed in favor of https://github.com/meilisearch/product/discussions/356 😇
